notes/fintech-incrementalism-and-responsible-innovation.md

```diff
@@ -2,31 +2,45 @@
 
-# Episode Notes
+Crypto is just the evolution of this trend of good incremental fintech improvemments. We want to continue moving the economy into a hyperfinanicalized 24/7 real-time always-trading market with even more complexity and lower friction than what we have presently. Blockchain can get us there. 
 
-## Summary
+Blockchain can be a vehicle for increase in financialization through the development of more complex, [blockchain](../concepts/blockchain.md)-based [financial products](../concepts/financial-asset.md). This ranges from better payment rails i.e. a better Visa, Stripe etc, to more efficient clearing systems, to full-scale innovation in financial engineering.
 
-In this episode, Rufus and Stephen  explore an argument for blockchain and crypto that they term “fintech incrementalism”. 
+Banks provide a clear example of what a lack of innovation looks like: banks are outdated and slow to innovate; core banking software is mostly from the 1980s; financial infrastructure is massively outdated in America and other coutrnies and the state of bit rot is vast. 
 
-This position assumes the capitalist notion that greater [financialization](../claims/is-hyperfinancialization.md) is an engine for progress, and claims that blockchain can be a vehicle for this increase in financialization through the development of more complex, [blockchain](../concepts/blockchain.md)-based [financial products](../concepts/financial-asset.md) – and the added market efficiency which will result. This ranges from better payment rails i.e. a better Visa, Stripe etc, to more efficient clearing systems, to full-scale innovation in financial engineering.
+Crypto is the means to fix all this. In the origial Bitcoin whitepaper Satoshi argues Bitcoin will provide a better payment rail, that we will be able to transfer money to people quickly and easily.
 
-The political imaginaries of this position are: more efficient markets and more bespoke and customized financial products to buy. [Capitalism](../concepts/capitalism.md) is net positive in the world, and a fairer, more efficient and transparent economy benefits everyone. 
+In addition, the future of finance will see private money and [Central Bank Digital Currencies](../concepts/cbdc.md) coexist and compete for market share. A global distributed ledger will provide greater market transparency and efficiency of sovereign flows.
 
-Rufus and Stephen 'steel man' the “fintech incrementalism” position, identifying 7 key claims, before then conducting an analysis of the position.
+While it’s common to have a knee-jerk reaction to financial complexity, we need to take things on a case-by-case basis rather than blanket write off financialization. Regulation can catch up with technical innovation.
+
+## Evidence of the claim being made
+
+## Evaluation: False
```

Comment:
Generally very good.

Main point is that there are very clearly two or three distinct claims being evaluated here - in which case let's make that explicit in both thesis and evaluation.